test-data/unit/check-incremental.test

```diff
@@ -5389,3 +5393,58 @@ E = Enum('E', f())  # type: ignore
 [builtins fixtures/list.pyi]
 [out]
 [out2]
+
+[case testChangedPluginsInvalidateCache]
```

Comment:
Maybe add another test where only the version changes (e.g. import version from another module so that the text of the plugin doesn't change).